clients/public/src/components/Login.tsx

```diff
@@ -126,6 +127,7 @@ function Login() {
                                     required
                                 />
                             </div>
+                            {ForgotPasswordMessage && <div className="forgot-password-message">{ForgotPasswordMessage}</div>}
```

Comment:
react'da {variable && <element>sth to show</element>} conditional rendering kullanımı önerilmiyor, onun yerine 
```{forgotPasswordMessage ? <div className="forgot-password-message">{forgotPasswordMessage}</div> : null}```
şeklinde ternary operator kullanımını tercih edersen daha iyi olacaktır, {check ? render sth : null} kullanımından bahsettim